Implemented camera switching functionality on main page.

TheImperialOne opened the pull request:
Addressing issue #25, we can switch cameras on the main page.  The main page also announces toast messages when switching cameras. However, the user cannot switch through cameras during recording yet. I am planning to implement that in the future. 
Please consider this for a review.

anonfaded commented:
Great work. 
But to understand, can you explain why you removed these import lines?
```java
import static com.fadcam.ui.SettingsFragment.PREF_LOCATION_DATA;
import com.arthenica.ffmpegkit.ReturnCode;
import android.view.HapticFeedbackConstants;
import android.view.MotionEvent;

import android.location.Location;
import android.location.LocationListener;
import android.location.LocationManager;
import android.os.Bundle;

import org.json.JSONArray;
import org.json.JSONObject;
import java.util.concurrent.ExecutorService;
```
Also need some enhancements:
- [ ] Due to limited space and some small screen devices, we should make the switch button to be in circular shape and only with the icon, no 'switch' label.

For reference see how it looks on smaller screen:
![image](https://github.com/user-attachments/assets/432ac18b-31a8-4216-af9d-5d6f3e47e3c7)


- [ ] Also please remove the `.idea` folder from the Pull Request as those files are unnecessary.

TheImperialOne commented:
To address the issues mentioned above:
1] Some import statements were unused, so I removed them to optimize memory consumption
2] The issue with the switch button has been addressed
And I added a new enhancement, I synced the camera switching status between the main page and the settings fragment!
I also added a new toast message to announce the camera selected when the activity is launched for the first time.

anonfaded commented:
> To address the issues mentioned above: 1] Some import statements were unused, so I removed them to optimize memory consumption 2] The issue with the switch button has been addressed And I added a new enhancement, I synced the camera switching status between the main page and the settings fragment! I also added a new toast message to announce the camera selected when the activity is launched for the first time.

Alright will review in some time. Btw i was using the location and haptic feedback imports bcz on button press the haptic feedback works, and the location preference was there to update in real time when user select the `Enable location data in watermark`, but in recent commits that i did i think something went wrong and now that location preference doesn't update in real time, it works when i close the app and reopen. But i will try to find the issue until then I'll see if without those imports the app functions correctly, mainly the `haptic feedback` and the `location data in watermark`.
